components/pagination/_pagination.scss

```diff
@@ -0,0 +1,108 @@
+/*
+_pagination.scss
+*/
+$pagingationSize: 2.5rem;
+.pagination {
+  width: 100%;
+  clear: both;
+}
+.pagination__items {
+  padding: 0;
+}
+.pagination__item {
+  width: $pagingationSize;
+  height: $pagingationSize;
+  line-height: 2.3rem;
+  text-align: center;
+  float: left;
+  margin-right: 0.25rem;
+  box-sizing: border-box;
+
+}
+.pagination__item_current {
+  & .pagination__link {
+    background-color: $color-secondary;
+    color: $color-white;
+    text-decoration: none;
+    /* Should these be made a mixin? */
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
+    /* End ? */
+
+    border-radius: 3px;
+    border: 0.16667rem solid $color-secondary;
```

Comment:
Don't think this needs a border?

Reply:
Yeah this does to stop the current page indicator vanishing like buttons did for Zoomtext with inverted colours. The background colour is removed as well as part of the inversion so border is added in the same colour as background as this is unaffected.